sycl/include/sycl/context.hpp

```diff
@@ -260,6 +260,35 @@ class __SYCL_EXPORT context : public detail::OwnerLessBase<context> {
   friend T detail::createSyclObjFromImpl(decltype(T::impl) ImplObj);
 };
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+// context.hpp depends on exception.hpp but we can't define these ctors in
+// exception.hpp while context is still an incomplete type.
```

Comment:
Can we leave a similar comment pointing to context.hpp in exception.hpp?

Reply:
Done. Thanks.